feat: hubble async push (#5129)
jina-ai/jina: docs/fundamentals/executor/hub/query-executor.md

(query-executor)=

> # Query

Reviewer: ```suggestion
## Building status of an Executor 
```

> # Query

Reviewer: This should not be its own session but rather be part of the Publish executor part

Reviewer 2: Exactly, should be part of `push-executor`.

> If you want to query your pushed executor build progress, you can query it.

Reviewer: ```suggestion
You can query the build status of a pushed Executor by doing:
```

> ## Use task id query the executor build progress 

Reviewer: lets remove this subsection

```bash
jina hub query [--verbose/--replay] <async upload executor task_id>

> ```

Reviewer: what is the difference between --verbose and --replay ? It should be explained and there should be two example and not one
